third_party/2and3/requests/models.pyi

```diff
@@ -127,7 +127,10 @@ class Response:
     def apparent_encoding(self) -> str: ...
     def iter_content(self, chunk_size: Optional[int] = ...,
                      decode_unicode: bool = ...) -> Iterator[Any]: ...
-    def iter_lines(self, chunk_size=..., decode_unicode=..., delimiter=...): ...
+    def iter_lines(self,
+                   chunk_size: Optional[int] = ...,
+                   decode_unicode: bool = ...,
+                   delimiter: Optional[str] = ...) -> Iterator[Any]: ...
```

Comment:
This should be `Union[Text, bytes]]` I think (the required type depends on the value of `decode_unicode`).